library/ui/src/main/java/com/google/android/exoplayer2/ui/PlayerView.java

```diff
@@ -1055,26 +1070,20 @@ private boolean setArtworkFromMetadata(Metadata metadata) {
       if (metadataEntry instanceof ApicFrame) {
         byte[] bitmapData = ((ApicFrame) metadataEntry).pictureData;
         Bitmap bitmap = BitmapFactory.decodeByteArray(bitmapData, 0, bitmapData.length);
-        return setArtworkFromBitmap(bitmap);
+        return setDrawableArtwork(new BitmapDrawable(getResources(), bitmap));
       }
     }
     return false;
   }
 
-  private boolean setArtworkFromBitmap(Bitmap bitmap) {
-    if (bitmap != null) {
-      int bitmapWidth = bitmap.getWidth();
-      int bitmapHeight = bitmap.getHeight();
-      if (bitmapWidth > 0 && bitmapHeight > 0) {
-        if (contentFrame != null) {
-          contentFrame.setAspectRatio((float) bitmapWidth / bitmapHeight);
-        }
-        artworkView.setImageBitmap(bitmap);
-        artworkView.setVisibility(VISIBLE);
-        return true;
+  private boolean setDrawableArtwork(Drawable drawable) {
+    if(drawable != null) {
+      artworkView.setImageDrawable(drawable);
```

Comment:
We need to add `artworkView.setVisibility(VISIBLE)` here for the artwork to actually be displayed.